pipeline/backend/kubernetes/pod.go

```diff
@@ -464,6 +480,15 @@ func mapToEnvVars(m map[string]string) []v1.EnvVar {
 	return ev
 }
 
+func isRepoTrusted(step *types.Step) bool {
+	// TODO: probably, should be stored in the model (pipeline/backend/types/config.go)
+	trustedRepo, err := strconv.ParseBool(step.Environment["CI_REPO_TRUSTED"])
```

Comment:
Is it safe to read it from the env? It can't be overwritten by users?

Reply:
> Is it safe to read it from the env?

¯\_(ツ)_/¯
Elaborate, please.
Do you suggest to use dedicated variable in `pipeline/backend/types/config.go`?
What is the difference between env var (one map's entry) vs dedicated variable?

> It can't be overwritten by users?

Seems, not for now, at least: #3164.

Reply:
>  Elaborate, please.

Right now I have not suggested anything :) I just asked if it is safe to read a security-related setting from an env var. If only the server can set it, and it can't be overwritten by the pipeline config, I'm fine with it, but it might increase the possibilities to become an issue in the future.

Reply:
Yes, might be an issue. Depends on #3164 implementation.
So, what are we gonna do?

Reply:
IMO, the clean solution would be to pass down the repo settings to the backend similar to `Environment` e.g. 
`Step.RepoSettings`.